spec/decorators/content_items_decorator_spec.rb

```diff
@@ -4,7 +4,7 @@
   include Draper::ViewHelpers
 
   let(:organisation) { create(:organisation, title: "Organisation title", slug: "the-slug") }
-  let(:taxonomy) { create(:taxonomy, title: "taxonomy a") }
+  let(:taxonomy) { create(:taxonomy, title: "taxonomy a", content_id: "content-id-for-taxon123") }
 
   context "without filter params" do
     it "renders the default page title" do
```

Comment:
Can remove `allow(helpers)..` here and the import of `include Draper::ViewHelpers` as they're no longer used.